insertion_sort/insertion_sort.py

```diff
@@ -0,0 +1,22 @@
+def insertionSort(alist):
+    """
+    Performs a insertion sort
+    :type alist: object
+    """
+    for index in range(1,len(alist)):
+        currentvalue = alist[index]
+        position = index
+    while position>0 and alist[position-1]>currentvalue:
+         alist[position] = alist[position-1]
```

Comment:
You seem to have indentation errors, please check it once using pep8 (linting tool)